tools/benchmark/cmd/watch.go

```diff
@@ -137,8 +137,8 @@ func watchFunc(cmd *cobra.Command, args []string) {
 	<-pdoneC
 
 	// put phase
-	// total number of puts * number of watchers on each key
-	eventsTotal = watchPutTotal * (watchTotal / watchedKeyTotal)
+	// total number of puts * (number of watchers on each key) / (number of events triggered per put)
+	eventsTotal = watchPutTotal * (watchTotal / watchedKeyTotal) / (watchedKeyTotal / watchKeySpaceSize)
```

Comment:
I was under the impression that `eventsTotal` is the expected total number of watch events?

The `putreqc <- v3.OpPut(watched[i%(len(watched)])` loop should probably use `watchPutTotal` instead of `eventsTotal`. If it uses `watchPutTotal` as the number of puts, `eventsTotal` could be computed like:
```go
eventsTotal := 0
for i := 0; i < watchPutTotal; i++ {
    eventsTotal += numWatchers[watched[i%len(watched)]]
}
```